recipes/future/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = "0.15.2" %}
+
+package:
+  name: future
+  version: {{ version }}
+
+source:
+  fn: future-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/source/f/future/future-{{ version }}.tar.gz
+  sha256: 3d3b193f20ca62ba7d8782589922878820d0a023b885882deec830adbf639b97
+
+build:
+  number: 1
```

Comment:
There is already a build 0 of 0.15.2 in the main conda recipes at ContinuumIO/anaconda-recipes/tree/master/future. Should I still use build 0?